calls/forms.py

```diff
@@ -14,6 +19,27 @@
 CALLS_MAX_DISTANCE_MILES = settings.CALLS_MAX_DISTANCE_MILES
 CALLS_MAX_LIST_SIZE = settings.CALLS_MAX_LIST_SIZE
 
+class CommaSeparatedCallersTextArea(Widget):
+    """
+    Transform many-to-many callers into comma separated values for display.
+    """
+    def render(self, name, value, attrs=None, renderer=None):
+        final_attrs = self.build_attrs(attrs, {'type':'text', 'name':name})
+        caller_emails = []
+
+        if value is not None:
+            """Loop through the caller_ids we have stored on the many-to-many
+            relationship and get the corresponding object"""
+            for caller_id in value:
+                callprofile = CallProfile.objects.get(pk=caller_id)
```

Comment:
I don't think a form field widget render function should be creating new records in db. Doesn't this violate the purpose of the render method?
```
render(name, value, attrs=None, renderer=None)[source]¶
Renders a widget to HTML using the given renderer. If renderer is None, the renderer from the FORM_RENDERER setting is used.
```
https://docs.djangoproject.com/en/1.11/ref/forms/widgets/#django.forms.Widget.render

Each piece of the software app has a specific purpose. Some are for creating db records. Some are for rendering html. Please think about what type of logic is happening and where it should be located based on the overall design pattern and separation of concerns principle: https://en.wikipedia.org/wiki/Separation_of_concerns